scripts/publish.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Publish an updated version of pileup.js on NPM.
+
+set +x
+set -o errexit
+
+# Ensure a clean dist directory
+rm -rf dist
+
+npm run jstransform
+npm run browserify
+npm run uglify
```

Comment:
if you'd like, you can shorten these three runs as: `npm run build` (up to you, not really necessary)